webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path");
+
+module.exports = {
+  entry: "./src/index.js",
+  output: {
+    filename: "main.js",
+    path: path.resolve(__dirname, "dist")
+  },
+  watch: true,
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: ["style-loader", "css-loader"]
+      },
+      {
+        test: /\.(png|svg|jpg|gif)$/,
+        use: ["file-loader"]
+      }
+    ]
+  }
+};
+
+const BrowserSyncPlugin = require("browser-sync-webpack-plugin");
+
+module.exports = {
+  //
+  watch: true,
+  plugins: [
+    new BrowserSyncPlugin({
+      // browse to http://localhost:3000/ during development,
```

Comment:
**You can clear up your code by removing comments, you can always use git history to access previous code right :slightly_smiling_face:**